docs/_includes/scripts.html

```diff
@@ -0,0 +1,22 @@
+{% if page.mathjax %}
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    tex2jax: {
+      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+      processEscapes: true
+    }
+  });
+</script>
+<script
+  type="text/javascript"
+  charset="utf-8"
+  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"
+>
+</script>
+<script
+  type="text/javascript"
+  charset="utf-8"
+  src="https://vincenttam.github.io/javascripts/MathJaxLocal.js"
```

Comment:
I can not see this file had been used anywhere.

Does this mean that this is a theme system file? if so, could you please make sure that you did not overwrite other functions?

Reply:
I mean, the `docs/_includes/scripts.html`